techmatters/terraso-mobile-client
# feat: create privacy bottom sheet

## Description
Adds an InfoModal bottom sheet for privacy info on various screens.

### Related Issues
Fixes #586

### Verification steps
1. When creating a site, tap the info (i) button to open the privacy bottom sheet.
2. View the main site dashboard on a created site. If no project has been assigned, you should see radio buttons for privacy. Tap the info button to open the privacy bottom sheet.
3. When creating a project, tap the info button to open the sheet.
4. View the inputs tab on a created project. You should see privacy radio buttons with an info button that opens a bottom sheet.
